airflow/utils/docs.py

```diff
@@ -27,7 +27,7 @@ def get_docs_url(page: Optional[str] = None) -> str:
             "http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow/latest/"
         )
     else:
-        result = f'https://airflow.apache.org/docs/{version.version}/'
+        result = f'https://airflow.apache.org/docs/apache-airflow/{version.version}/'
```

Comment:
Maybe its worth updating this function to get URLs for different sections of the documentation for example https://airflow.apache.org/docs/apache-airflow-providers-google?

But also this isn't used in many places.

Reply:
It is not necessary. This feature is only used in Airflow core, so we can only support links to the core.